tools/influx_si/macros.xml

```diff
@@ -0,0 +1,265 @@
+              <param argument="--fseries" type="text" value="" label="--fseries" optional="true" help="File name with free parameter values for multiple starting points. Default: &#x27;&#x27; (empty, i.e. only one starting point from the FTBL file is used)" />
+              <param argument="--iseries" type="text" value="" label="--iseries" optional="true" help="Indexes of starting points to use. Format: &#x27;1:10&#x27; -- use only first ten starting points; &#x27;1,3&#x27; -- use the the first and third starting points; &#x27;1:10,15,91:100&#x27; -- a mix of both formats is allowed. Default: &#x27;&#x27; (empty, i.e. all provided starting points are used)" />
+              <param argument="--seed" type="integer" min="0" value="" label="--seed" optional="true" help="Integer (preferably a prime integer) used for reproducible random number generating. It makes reproducible random starting points (--irand) but also Monte-Carlo simulations for sensitivity analysis. Default: none, i.e. current system value is used, so random drawing will be varying at each run." />
+              <param argument="--excl_outliers" type="float" min="0" max="1" value="" label="--excl_outliers" optional="true" help="This option takes an optional argument, a p-value between 0 and 1 which is used to filter out measurement outliers. The filtering is based on Z statistics calculated on reduced residual distribution. Default: 0.01." />
+              <param argument="--nocalc" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--nocalc" optional="true" help="generate an R code but not execute it." />
+              <param argument="--addnoise" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--addnoise" optional="true" help="Add centered gaussian noise to simulated measurements written to _res.kvh file. SD of this noise is taken from FTBL file" />
+              <param argument="--TIMEIT" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--TIMEIT" optional="true" help="developer option: measure cpu time or not" />
+              <param argument="--prof" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--prof" optional="true" help="developer option: do time profiling or not" />
+              <param argument="--tblimit" type="integer" min="0" value="0" label="--tblimit" optional="true" help="developer option: set trace back limit for python error messages" />
+            </section>
+        </inputs>
+    </xml>
+    <xml name="help">
+        <help><![CDATA[
+        Usage: influx_s [options] /path/to/FTBL_file1 [FTBL_file2 [...]]
+
+Optimize free fluxes and optionaly metabolite concentrations of a given static
+metabolic network defined in an FTBL file to fit 13C data provided in the same
+FTBL file.
+
+Options:
```

Comment:
those options are already described in the param section as label and as help ... so it could be removed here if you like